docs/api/app.md

```diff
@@ -1325,6 +1322,7 @@ Returns `Object`:
   * `enabled` boolean (optional) _Windows_ - `true` will change the startup approved registry key and `enable / disable` the App in Task Manager and Windows Settings.
     Defaults to `true`.
   * `name` string (optional) _Windows_ - value name to write into registry. Defaults to the app's AppUserModelId().
```

Comment:
Looks like this property is now available on macOS, so it should be documented here.